src/many-migration/src/lib.rs

```diff
@@ -554,7 +554,6 @@ impl<'a, T, E, IDX: AsRef<str>> Index<IDX> for MigrationSet<'a, T, E> {
 }
 
 /// Kept for backward compatibility.
-#[deprecated = "Should use MigrationSet::load() instead."]
```

Comment:
These warnings were triggering on builds.  Either these should be removed, or they should be re-added _along with_ the remedies for the places where they are currently being used.